src/main/java/com/github/jreddit/parser/listing/CommentsListingParser.java

```diff
@@ -0,0 +1,61 @@
+package com.github.jreddit.parser.listing;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.json.simple.parser.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.github.jreddit.parser.entity.Comment;
+import com.github.jreddit.parser.entity.Thing;
+import com.github.jreddit.request.error.RedditError;
+
+public class CommentsListingParser extends RedditListingParser {
+
+	/** Logger for this class. */
+	final static Logger LOGGER = LoggerFactory.getLogger(CommentsListingParser.class);
+	
+	public CommentsListingParser() {
```

Comment:
I think if you ditch the default constructor declaration, the compiler will create automatically a default consutrctor and will call the super method as well.